pulseguardian/web.py

```diff
@@ -491,99 +484,9 @@ def bindings_listing(queue_name):
     return jsonify({"queue_name": queue_name, "bindings": bindings})
 
 
-# Authentication related
-
-@app.route('/auth/callback')
-@sh.wrapper()
-def callback_handling():
-    """
-    Callback from Auth0
-
-    Auth0 will call back to this endpoint once it has acquired a user in some
-    capacity (from Google, or their email).  Here we verify that what we got
-    back is consistent with Auth0, then match that against our internal DB and
-    either find the user, or create it.
-    """
-
-    # Validate what we got in the request against the Auth0 backend.
-    code = request.args.get('code')
-    json_header = {'content-type': 'application/json'}
-    token_url = "https://{domain}/oauth/token".format(
-        domain=config.auth0_domain
-    )
-    token_payload = {
-        'client_id': config.auth0_client_id,
-        'client_secret': config.auth0_client_secret,
-        'redirect_uri': config.auth0_callback_url,
-        'code': code,
-        'grant_type': 'authorization_code',
-    }
-    token_info = requests.post(token_url,
-                               data=json.dumps(token_payload),
```

Comment:
./pulseguardian/web.py:34:1: F401 'json' imported but unused

(Also fwiw, the `json.dumps()` here was unnecessary - requests has a `json` argument that does this automatically)